From 7c163456d39d5644ff4b50adfdd7797a063c8ba6 Mon Sep 17 00:00:00 2001
From: Yuyao Lin <linyuyao1@huawei.com>
Date: Wed, 9 Nov 2022 10:50:47 +0800
Subject: [PATCH 129/905] [Huawei] misc: djtag: optimize the comment format for
 hisi-djtag

Offering: HULK
hulk inclusion
category: cleanup
bugzilla: 187927

--------------------------------

Keep comments in a uniform format.

Signed-off-by: Yuyao Lin <linyuyao1@huawei.com>
---
 drivers/misc/hisi-djtag/hisi-djtag.c | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/drivers/misc/hisi-djtag/hisi-djtag.c b/drivers/misc/hisi-djtag/hisi-djtag.c
index 6dd64363bdaf..f516e4a307e2 100644
--- a/drivers/misc/hisi-djtag/hisi-djtag.c
+++ b/drivers/misc/hisi-djtag/hisi-djtag.c
@@ -27,12 +27,17 @@
 
 #include <asm/cacheflush.h>
 
-#define SC_DJTAG_TIMEOUT		1000000	/* step: udelay(1) 1s */
-#define SC_DJTAG_RESET			80000	/* step: udelay(100) 8s */
+/* step: udelay(1) 1s */
+#define SC_DJTAG_TIMEOUT		1000000
+
+/* step: udelay(100) 8s */
+#define SC_DJTAG_RESET			80000
 
 /* for 660 and 1382 totem */
 #define SC_DJTAG_MSTR_EN		0x6800
-#define DJTAG_NOR_CFG			BIT(1)	/* accelerate R,W */
+
+/* accelerate R,W */
+#define DJTAG_NOR_CFG			BIT(1)
 #define DJTAG_MSTR_EN			BIT(0)
 #define SC_DJTAG_MSTR_START_EN		0x6804
 #define DJTAG_MSTR_START_EN		0x1
-- 
2.17.1

